lp-builder-config/ceph.yaml

```diff
@@ -58,6 +58,21 @@ projects:
     launchpad: charm-ceph-mon
     repository: https://opendev.org/openstack/charm-ceph-mon.git
 
+  - name: Ceph NFS Charm
+    charmhub: ceph-nfs
+    launchpad: charm-ceph-nfs
+    repository: https://opendev.org/openstack/charm-ceph-nfs.git
+    branches:
+      master:
+        channels:
+          - latest/edge
+      stable/quincy:
```

Comment:
just a comment, since those stable branches don't exist, they would produce recipes associated to missing branches hence not triggering builds until those branches are really created, this is a nice side effect of this design.